index.js

```diff
@@ -0,0 +1,3 @@
+const rojoSays = require('./lib/rojo-says');
+
+console.log(rojoSays('what colour are you?'));
```

Comment:
This is actually an example of a manual tests (_you_ read it yourself to test the outcome), good ol' automated tests do this for us.